style.css

```diff
@@ -52,3 +58,26 @@ a {
 .card {
   max-width: none;
 }
+
+.question-data {
+  display: flex;
+  justify-content: space-between;
```

Comment:
If you have created utilities classes in your component library then you can use them there otherwise it's fine!

Reply:
i don't have utility classes for flex properties, thank you for your time.